migtools/oadp-non-admin
# fix: NonAdminBackup cluster validation

## Why the changes were made

Improve user experience by blocking creation of wrong NAB CR, instead of erroring out in Reconcile.

Fix https://github.com/migtools/oadp-non-admin/issues/80

Blocked by `TODO open PR in OADP updating NAB CRD`

## How to test the changes made

Try to create a NAB without `spec.backupSpec` in a cluster. It should error out and not allow creation saying `spec.backupSpec` is required.
